src/components/team-member.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faLinkedinIn } from '@fortawesome/free-brands-svg-icons';
+
+const TeamMember = (props) => {
```

Comment:
Why was the name of the component file changed?

Reply:
Karen suggested that we use kebab-case for the components name. I'm not sure why, but maybe MUI uses kebab-case too?